iao.owl

```diff
@@ -7242,6 +7388,18 @@ No imports</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/NCBITaxon_9606 -->
```

Comment:
consider adding this in the imports

Reply:
Hi @jmwhorton, if you are not sure how to import OBO terms to IAO in the import-OBO.owl file, I can help with it.

Reply:
Thank you, @zhengj2007.  I believe I have figured it out.

Reply:
Both 'organism' and 'Homo sapiens' need go to the import-OBO.owl. The IAO using ontoFox to extract the terms from external ontologies. Please add the terms in OntoFox-input.txt and run ontoFox tool to generate the import-OBO.owl.

Reply:
Added import to OntoFox-input.txt